tests/functional/api/error_api_test.c

```diff
@@ -12,6 +12,22 @@
 
 MTF_BEGIN_UTEST_COLLECTION(error_api_test)
 
+MTF_DEFINE_UTEST(error_api_test, err_ctx)
+{
+    unsigned int matches = 0;
+
+    for (int i = HSE_ERR_CTX_BASE; i < HSE_ERR_CTX_MAX; i++) {
+        switch (i) {
+        case HSE_ERR_CTX_NONE:
+        case HSE_ERR_CTX_TXN_EXPIRED:
+            matches++;
+            break;
+        }
+    }
+
+    ASSERT_EQ(HSE_ERR_CTX_MAX - HSE_ERR_CTX_BASE, matches);
+}
+
```

Comment:
You should use 'for (enum hse_err_ctx i = ...)' because the enum's underlying type might not be "int".

Reply:
Good point. C annoyances...